models/interlibrary_loan_requests.rb

```diff
@@ -0,0 +1,54 @@
+class InterlibraryLoanRequests
+  def initialize(parsed_response:)
+    @parsed_response = parsed_response
+    @requests = parsed_response.select { |request| request["ProcessType"] != "DocDel" && request["TransactionStatus"] != "Request Finished" }
+  end
+
+  def count
+    @requests.length
+  end
+
+  def each(&block)
+    @requests.each do |request|
+      @request = InterlibraryLoanRequest.new(request)
```

Comment:
I would make @request be a list of InterlibraryLoanRequest objects, rather than having that performed in the `each` method. You could add a `map` step to line 4 to transform the array or you can use the ruby 2.7 method `filter_map` to do both in one step: https://ruby-doc.org/core-2.7.0/Enumerable.html#method-i-filter_map